Formula/tfel.rb

```diff
@@ -0,0 +1,63 @@
+class Tfel < Formula
+  desc "Code generation tool dedicated to material knowledge for numerical mechanics"
+  homepage "https://thelfer.github.io/tfel/web/index.html"
+  url "https://github.com/thelfer/tfel/archive/refs/tags/TFEL-4.0.tar.gz"
+  sha256 "ed3fb2f59c6b8c9896606ef92276f81942433dd5f60d8130ba07c3af80b039e2"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/tfel.git", using: :git, branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.9" => :build
```

Comment:
Does it need a specific version of Python? If not, `uses_from_macos "python" => :build`.

Reply:
@carlocab If I use `uses_from_macos "python" => :build`, my build fails on `arm-64`: <https://github.com/Homebrew/homebrew-core/runs/7771212216?check_suite_focus=true>. Any idea why ? Also, does that work on `linux` ?

Reply:
No idea, but it's worth trying `python@3.10` now, since we made that the linked Python formula recently. Though, if you just want a Python interpreter, it might be enough to do
```ruby
uses_from_macos "python" => :build
```

Reply:
Oops, sorry, copy-paste error. I meant we should try
```ruby
depends_on "python@3.10" => :build
```